client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -414,19 +413,8 @@ const RecurringPaymentsPlanAddEditModal = ( {
 				</SectionNavTabs>
 			</SectionNav>
 			<div className="memberships__dialog-sections">
-				<div
-					className={ classnames( 'memberships__dialog-section', {
-						'is-visible': currentDialogTab === TAB_GENERAL,
-					} ) }
-				>
-					{ renderGeneralTab() }
-				</div>
-				<div
-					className={ classnames( 'memberships__dialog-section', {
-						'is-visible': currentDialogTab === TAB_EMAIL,
-					} ) }
-				>
-					{ renderEmailTab() }
+				<div className="memberships__dialog-section is-visible">
```

Comment:
The CSS hiding seems hacky and root of some of the problems, I think it's best to remove `is-visible` and untangle this:

https://github.com/Automattic/wp-calypso/blob/afed935719c5c68a2bf5f756386379dedc2d967d/client/my-sites/earn/memberships/style.scss#L316-L318

https://github.com/Automattic/wp-calypso/blob/afed935719c5c68a2bf5f756386379dedc2d967d/client/assets/stylesheets/shared/functions/_z-index.scss#L230

Reply:
> I think it's best to remove is-visible and untangle this

Yes.

> The CSS hiding seems hacky 

Unfortunately, that is pretty much how Earn was made, apparently. We have a lot of work to bring every up to a certain standard...

Reply:
Hey @lezama @simison @millerf - I took a look at this. There seems to be a bit of unusual css around these tabs. I found what I think could be a very simple solution. This solution doesn't wrestle with any of the underlying complexities here, so it's not a 'thorough' solution. It just bumps the z-index from 1 to 2.

But best I can tell, this works, is simple, and should not affect anything else:
https://github.com/Automattic/wp-calypso/pull/76840/files